gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -463,6 +464,15 @@ type gcsDir struct {
 	CommonPrefixes []Prefix
 }
 
+// getStaticDir returns KO_DATA_PATH if the image is built using ko, as that's
+// where static files are placed in the image, otherwise this should be empty,
+// and the static files are placed at root level in this case. For example:
+// - for ko: /var/run/ko/styles/style.css
+// - otherwise: /styles/style.css
+func getStaticFilePath(rel string) string {
```

Comment:
Instead of doing this dance, can we just use [embed](https://pkg.go.dev/embed) so all of this gets embedded into the binary at compile time?

Reply:
First time heard about `embed`, but it looks very interesting and I think it's going to work, let me play with it. Will need to figure out how to do a e2e test manually first

Reply:
Unfortunately I don't think it'll work. The problem is that `embed` embeds static files during compilation, which doesn't work with static dirs that are configurable via cmd args, I have commented the line in the other file.

However I feel it would work for deck, where we can embed all static files during compilation, then deck can be started with just `go run`. For achieving this we'll need to source control rolled up typescript bundles, which should be fine